### DISCUSSION TITLE
Error communicating with OpenAI怎么解决

### DISCUSSION THREAD

**Discussion by Ley-zhang** [kaixindelele/ChatPaper#165] [Q&A]
主要报错如下：
summary_error: Error communicating with OpenAI: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/chat/completions (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)')))

**Comment by mocusez** [comment]
挂梯子
或者用国内的反向代理
